packages/eas-cli/src/submit/ArchiveSource.ts

```diff
@@ -317,8 +317,16 @@ async function handleBuildListSourceAsync(
 }
 
 function formatBuildChoice(build: BuildFragment, expiryDate: Date): prompts.Choice {
-  const { id, updatedAt, runtimeVersion, buildProfile, gitCommitHash, gitCommitMessage, channel } =
-    build;
+  const {
+    id,
+    updatedAt,
+    runtimeVersion,
+    buildProfile,
+    gitCommitHash,
+    gitCommitMessage,
+    channel,
+    message,
+  } = build;
 
   const formatValue = (field?: string | null): string => (field ? chalk.bold(field) : 'Unknown');
```

Comment:
Should we actually display "Unknown" if some field is not defined? Maybe it's better not to display it?

Reply:
Also, it's worth noting that the logic for displaying `message` is not consistent with this.

Reply:
>Should we actually display "Unknown" if some field is not defined? Maybe it's better not to display it?
@dsokal What's your opinion?

Reply:
I'd slightly prefer not displaying values when they are not available. But I'm not feeling strongly about it. What matters is that this should be consistent.

Reply:
What do you think about it now?